client/js/hah-app/reducers.js

```diff
@@ -66,10 +66,7 @@ const localStateReducer = (state = {}, action) => {
 				verificationPortalFormStatus: action.formStatus,
 			}
 		default:
-			return {
-				...state,
-				verificationPortalFormStatus: 'Pending',
-			}
+			return state
```

Comment:
Setting the form status to "Pending" by default was causing issues with the refactor. Also, it's best practice to just return the `state` as-is by default in reducers.